nodejs/json-rpc-relay/integration.tests/consensus.nodes/eth.sendRawTransaction.test.js

```diff
@@ -0,0 +1,48 @@
+console.clear();
+
+const { ContractId } = require('@hashgraph/sdk');
+
+require('dotenv').config({path: '.env'});
+
+const { ethers } = require("hardhat");
+
+const operatorPrivateKey = process.env.OPERATOR_PRIVATE_KEY;
+const communityHashioTestnet = process.env.COMMUNITY_TESTNET_URL;
+const arkhiaJsonRpcRelayTestnet  = `${process.env.ARKHIA_TESTNET_URL}/${process.env.ARKHIA_API_KEY}`;
+
+describe('RPC', () => {
```

Comment:
RPC is a bit generic, as a reader not sure what i can deduct from this ,  is it better something more descriptive?

Reply:
Changed the name to `Deploy and Call Contract via sendRawTransaction`
This is done